Program.cs

```diff
@@ -1,10 +1,26 @@
-﻿namespace Training
-{
-    internal class Program
-    {
-        static void Main(string[] args)
-        {
-            Console.WriteLine("Hello, World!");
-        }
-    }
-}
\ No newline at end of file
+﻿// Convert decimal to hexadecimal and binary
+//string hexvalue = input.ToString ("X") ----> in-built function for reference
+
+Console.Write ("Enter a decimal value: ");
+if (int.TryParse (Console.ReadLine (), out int input)) {
+   //To hexadecimal
+   int input1 = input;
+   string[] hexdigits = { "0", "1", "2", "3", "4", "5", "6", "7", "8", "9", "A", "B", "C", "D", "E", "F" };
+   string hexvalue = "";
+   if (input == 0) hexvalue = "0";

```

Comment:
If input is zero here itself you can assign both binary and hex to zero no need to repeat this again for binary in line number19